python/bindings_se3.cpp

```diff
@@ -29,19 +29,36 @@ void wrap_SE3(py::module &m)
 
   SE3.def(py::init<const Scalar, const Scalar, const Scalar,
                    const Scalar, const Scalar, const Scalar>());
-  // SE3.def(py::init<const Translation&, const Quaternion&>());
+  SE3.def(py::init([](const SE3d::Translation& pos, const Eigen::Vector4d& quat) {
```

Comment:
```suggestion
  SE3.def(py::init([](const SE3d::Translation& pos, const Eigen::Matrix<Scalar, 4, 1>& quat) {
```

Reply:
Done in https://github.com/artivis/manif/pull/211/commits/8af875836ae55db0b0e71f2bca40b2bce2fa5537